google/cloud/spanner/sql_statement.h

```diff
@@ -103,8 +103,15 @@ class SqlStatement {
     return !(a == b);
   }
 
- private:
+  /**
+   * Outputs a string representation of the given @p stmt to the given @p os.
+   *
+   * @warning This is intended for debugging and human consumption only, not
+   *     machine consumption as the output format may change without notice.
```

Comment:
See the other comment about a comma after "consumption".

Reply:
Done